website/docs/r/auth_server_policy_rule.html.markdown

```diff
@@ -40,6 +40,8 @@ The following arguments are supported:
 
 * `priority` - (Required) Priority of the auth server policy rule.
 
+* `group_whitelist` - (Required) The list of group ids to whitelist. Can be set to group id or to the following: "EVERYONE".
```

Comment:
"EVERYONE" should be styled with inline code:  `"EVERYONE"`

Reply:
@lasektom I agree with @thekbb. Can you update the markdown like this?

```markdown
`group_whitelist` - (Required) The list of group ids to whitelist. Can be set to group id or to the following: `"EVERYONE"`.
```